tartiflette/types/helpers/get_typename.py

```diff
@@ -0,0 +1,12 @@
+def get_typename(raw):
```

Comment:
Thus, what happens if nothing satisfied?

Reply:
Classname will be used and if it is not a knowtype (not in schema.types list), declared type in the schema will be used.

This method is used in case the declared type in the schema need to overriden (I.E. you are in a Union or an Interface). If the type is not declared in the object returned by the resolver (as a DictKey or an attribute), the classname is returned.

Then the __typename resolver tries to find the get_typename in the known type list.
If it is known it is returned, if it is not known, the type declared in the schema is returned.